src/sentry/sentry_metrics/consumers/last_seen_updater.py

```diff
@@ -52,6 +53,44 @@ def should_drop(self, message: Message[KafkaPayload]) -> bool:
         return FetchType.DB_READ.value not in str(header_value)
 
 
+class KeepAliveMessageFilter(StreamMessageFilter[TPayload]):
+    """
+    A message filter that wraps another message filter, and ensures that at
```

Comment:
This only works if the filter is considered a performance optimization and not required for product functionality otherwise incorrect filtering will actually cause functional changes. Can you please confirm this is the case for the last seen updater and also add this warning to the docstring.

Reply:
yeah i noted that in the pr desc, but added it in the docstring now